README.md

```diff
@@ -1,25 +1,55 @@
-# V for Visual Studio Code
+# V Language support for Visual Studio Code
 
 [![Version](https://vsmarketplacebadge.apphb.com/version/0x9ef.vscode-vlang.svg)](https://marketplace.visualstudio.com/items?itemName=0x9ef.vscode-vlang)
 [![Installs](https://vsmarketplacebadge.apphb.com/installs/0x9ef.vscode-vlang.svg)](https://marketplace.visualstudio.com/items?itemName=0x9ef.vscode-vlang)
 [![Build Status](https://travis-ci.org/0x9ef/vscode-vlang.svg?branch=master)](https://travis-ci.org/0x9ef/vscode-vlang)
 
-[**V Language**](https://vlang.io) support extension for Visual Studio Code. Syntax highlighting and code snippets.
+Provides [V language](https://vlang.io) support for Visual Studio Code.
 
-<https://marketplace.visualstudio.com/items?itemName=0x9ef.vscode-vlang&ssr=false>
-
-## Features
-
-* Snippets
-* Syntax highlighting
-* Code formatting
+- [Preview](#preview)
+- [Features](#features)
+  - [IntelliSense](#intellisense)
+  - [Code Navigation](#code-navigation)
+  - [Code Editing](#code-editing)
+  - [Testing](#testing)
+  - [Debugging](#debugging)
+  - [Others](#others)
 
 ## Preview
 
 ![First demo screenshot](./images/demo1.PNG)
 ![Second demo screenshot](./images/demo2.PNG)
 
-## Using
+## Features
+
+### Intellisense
+
+> WIP
+
+### Code Navigation
+
+> WIP
```

Comment:
Remove this if it's a WIP; there's no point advertising features that don't exist yet.